### PR TITLE
LPS-199428 Add tests

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

File: modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+function a() {
```
Comment (ambrinchaudhary) [inline modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js:1]: What is this?

**Review by adolfopa — APPROVED** [review]

File: modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js (hunk 1 above)
Comment (boton) [reply modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js:1]: 🤔

Comment (robertoDiaz) [reply modules/apps/portal/portal-tika-test/src/testIntegration/resources/com/liferay/portal/tika/test/dependencies/test-x-matlab.js:1]: Please see the integration test.

I use this fake .js file for testing porpouses

**Comment by robertoDiaz** [comment]
ci:forward

**Comment by adolfopa** [comment]
@robertoDiaz Not sure what the SF error is about, but maybe renaming the js function would be enough.

**Comment by SylviaLuan** [comment]
Hi @robertoDiaz,

Found a issue that related to this pull request.
[com.liferay.portal.tika.test.TikaRawMetadataProcessorTest](https://testray.liferay.com/home/-/testray/case_results/2450097046)

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by SylviaLuan** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
Hi @adolfopa,

Found a issue that related to this pull request.
[com.liferay.portal.tika.test.TikaRawMetadataProcessorTest
](https://testray.liferay.com/home/-/testray/case_results/2452086238)

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
Seems that latest changes in the test-x.mathlab.js file returned a text/plain mime type instead of a application/javascript

**Comment by robertoDiaz** [comment]
Seems that current js file content is needed by the test, but breaks the SF

Analyzing

**Comment by robertoDiaz** [comment]
ci:reevaluate:1267401_11803

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:reevaluate:1377401_15318

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by adolfopa** [comment]
ci:forward:force

**Pull request closed without merging**